src/sdk/python/examples/functions/run.py

```diff
@@ -9,26 +9,26 @@
 ##############################################################################
 
 import random
-import os
 import asyncio
 import logging
 
 from iotea.core.util.logger import Logger
+from iotea.core.util.mqtt_client import MqttProtocolAdapter
+from iotea.core.protocol_gateway import ProtocolGateway
+
 logging.setLoggerClass(Logger)
 logging.getLogger().setLevel(logging.INFO)
 
 from iotea.core.util.time_ms import time_ms
 
-os.environ['MQTT_TOPIC_NS'] = 'iotea/'
 
 # pylint: disable=wrong-import-position
 from iotea.core.talent_func import FunctionTalent
-from iotea.core.talent import Talent
 from iotea.core.rules import OrRules, OpConstraint, Rule, Constraint
 
 class MathFunctions(FunctionTalent):
-    def __init__(self, connection_string):
-        super(MathFunctions, self).__init__('math', connection_string)
+    def __init__(self, protocol_gateway_config):
+        super(MathFunctions, self).__init__('math', protocol_gateway_config)
```

Comment:
I think it should be `super().__init__` (to be aligned with the other python examples)